qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -1266,6 +1273,164 @@
       }
     }
   },
+  "discoveryConfig": {
+    "features": {
+      "pageTitle": {
+        "enabled": false,
+        "text": "Publication Discovery"
+      },
+      "search": {
+        "searchBar": {
+          "enabled": true
+        }
+      },
+      "authorization": {
+        "enabled": true
+      }
+    },
+    "aggregations": [{
+        "name": "Publications",
+        "field": "accession_id",
+        "type": "count"
+      }
+    ],
+    "tagSelector": {
+      "title": "Tags by category"
+    },
+    "studyColumns": [{
+        "name": "Title",
+        "field": "title",
+        "contentType": "string",
+        "width": 200
+      },
+      {
+        "name": "Authors",
+        "field": "authors",
+        "contentType": "string",
+        "width": 300
+      },
+      {
+        "name": "Affiliations",
+        "field": "author_affiliations",
+        "contentType": "string",
+        "ellipsis": true
+      },
+      {
+        "name": "Journal",
+        "field": "journal",
+        "contentType": "string"
+      },
+      {
+        "name": "Publication URL",
+        "field": "publication_url",
+        "contentType": "string"
+      },
+      {
+        "name": "Condition",
+        "field": "condition",
+        "contentType": "string"
+      },
+      {
+        "name": "Websites",
+        "field": "websites",
+        "contentType": "string"
+      }
+    ],
+    "studyPreviewField": {
+      "name": "Abstract",
+      "field": "abstract",
+      "contentType": "string",
+      "includeName": true,
+      "includeIfNotAvailable": false
+    },
+    "studyPageFields": {
+      "header": {
+        "field": "title"
+      },
+      "fieldsToShow": [{
+          "groupName": "Publication Identifiers",
+          "includeName": false,
+          "fields": [
+          {
+            "name": "Authors",
+            "field": "authors",
```

Comment:
does the discovery page not have logic to capitalize names automatically?

Reply:
Let me check and get back to you.

Reply:
Oh! I missed to get back. Portal does do the auto capitalization.

Reply:
great, let's remove the `name` fields that are only doing that then

Reply:
Not really, I tried taking off the name fields, but that totally removed the column title in the `table view`

Reply:
The column fields that appear as column headers are all in ALL_CAPS. That's what I thought you meant.